src/App.css

```diff
@@ -13,7 +71,188 @@ nav ul {
   max-width: 85%;
   margin: 0 auto;
 }
+
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  margin-bottom: 20px;
+  border: 1px solid #e8e8e8;
+  padding: 30px;
+}
+
+.title-div li {
+  display: inline-block;
+  font-size: 14px;
+  color: #0290ff;
+  font-weight: lighter;
+  cursor: pointer;
+  font-family: 'Roboto Slab', serif;
+}
+
+.title-div button {
+  background-color: white;
+  border: none;
+  border-left: 2px solid #e8e8e8;
+  border-right: 2px solid #e8e8e8;
+  padding: 0 10px;
+  margin: 0 10px;
+  font-size: 14px;
+  color: #0290ff;
+  font-weight: lighter;
+  font-family: 'Roboto Slab', serif;
+  cursor: pointer;
+}
+
+.title {
+  margin-bottom: 8px;
+  font-family: 'Roboto Slab', serif;
+}
+
+.author {
+  font-family: 'Roboto Slab', serif;
+  margin-bottom: 15px;
+  font-size: 14px;
+  color: #0290ff;
+}
+
+.progress-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* width: 100%;
+  max-width: 200px; */
+  padding-right: 30px;
+  margin-right: 30px;
+
+  /* border: 1px solid chocolate; */
+}
+
+.progress-div {
+  /* position: absolute; */
+  border-radius: 50%;
+  width: 4.25rem;
+  height: 4.25rem;
+  border: 4px solid #307bbe;
+  border-left-color: #e8e8e8;
+  -webkit-transform: rotate(45deg);
+  transform: rotate(45deg);
+  margin-right: 15px;
+}
+
+.percentage {
+  font-family: "Montserrat", sans-serif;
+  color: #121212;
+  font-size: 2rem;
+}
+
+.completed {
+  font-size: 0.875rem;
+  opacity: 0.5;
+  font-family: "Montserrat", sans-serif;
+  color: #121212;
+}
+
+.chapter-div {
+  border-left: 2px solid #e8e8e8;
+  padding-left: 60px;
+}
+
+.chapter-label {
+  font-family: "Roboto Slab", serif;
+  font-weight: lighter;
+  color: #121212;
+  font-size: smaller;
+  margin-bottom: 14px;
+}
+
+.chapter {
+  font-family: "Roboto Slab", serif;
+  font-weight: 300;
+  color: #121212;
+  margin-bottom: 16px;
+}
+
+.update-btn {
+  cursor: pointer;
+  border: none;
+  background-color: #0290ff;
+  border-radius: 3px;
+  padding: 8px 24px;
+  font-family: "Roboto Slab", serif;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  color: #fff;
+}
+
+.form {
+  border-top: 5px solid #e8e8e8;
+  padding-top: 40px;
+  margin-top: 40px;
+}
+
+.form input {
+  padding: 12px;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+  font-size: 1rem;
+  letter-spacing: -0.15px;
+  color: #121212;
+  margin-top: 25px;
+}
+
+.form input:focus {
+  outline: 1px solid #0290ff;
+}
+
+.form-title {
+  font-size: 1.25rem;
+  font-weight: 700;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.title-input {
+  display: inline-flex;
+  width: 100%;
+  max-width: 500px;
+}
+
+.author-input {
+  display: inline-flex;
+  width: 100%;
+  max-width: 350px;
+  margin: 0 30px;
+}
+
+.form-btn {
+  cursor: pointer;
+  width: 20%;
+  padding: 12px;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
```

Comment:
- [ ] Kindly work on the form fields and the button to stretch and align with the cards just as I highlighted in the screenshot below so things look aligned properly.

**Your Design**

![image](https://user-images.githubusercontent.com/50941074/208230625-21df99f9-bacf-42b9-ae65-1d7ad8efa033.png)

**The Original Design**

![image](https://user-images.githubusercontent.com/50941074/208230816-b271ae2f-36ae-4a15-ae9f-5004490ac18f.png)